tests/Auth0/Listeners/CreateClientsTest.php

```diff
@@ -67,27 +67,44 @@ public function it_creates_a_new_client_in_every_configured_tenant(): void
             ->with($integrationId)
             ->willReturn($integration);
 
+        $clientIds = [Uuid::uuid4(),Uuid::uuid4(),Uuid::uuid4()];
+
         $this->httpClient->expects($this->exactly(6))
             ->method('sendRequest')
             ->willReturnOnConsecutiveCalls(
-                new Response(201, [], Json::encode(['client_id' => 'client-id-1', 'client_secret' => 'client-secret-1'])),
+                new Response(201, [], Json::encode(['id' => $clientIds[0], 'client_id' => 'client-id-1', 'client_secret' => 'client-secret-1'])),
```

Comment:
I wonder if the `sendRequest` uses the `id`?

Reply:
it doesn't, removed it